### ISSUE TITLE
how do training ?

### ISSUE THREAD

**Issue by PythonImageDeveloper** [qjadud1994/CRNN-Keras#3]
Hi,
Please give me more information about training and prepare dataset, how do apply labels for this ?
please put this walk-through steps.
thanks.

**Comment by qjadud1994** [comment]
Sorry. I confirmed it a little late.
I will update the training detail in the readme.
Please check.

**Comment by qjadud1994** [comment]
In my opinion, I think that there will be a lot of performance difference according to learning data.
If you have a lot of real license plate images, I think it will work well.

In addition, I know that scene text recognition models such as GRCNN, RARE, and FAN as well as CRNN works well in real environment.

**Comment by zeynali** [comment]
GRCNN is CNN + GRU?

**Comment by qjadud1994** [comment]
No! 
GRCNN is Gated Recurrent Convolution Neural Network for OCR

t is similar to CRNN, but there is a difference in that certain parts of the CNN layer have been replaced by Gated Recurrent Convolution Layer.

[GRCNN](https://papers.nips.cc/paper/6637-gated-recurrent-convolution-neural-network-for-ocr)
